### DISCUSSION TITLE
Hilfe mit der Installation -- Konfiguration der Integration hängt

### DISCUSSION THREAD

**Discussion by MrSplit** [SukramJ/aiohomematic#748] [General]
Hallo,
Ich habe ein Home Assistant auf einem Intel NUC laufen und möchte meine HomeMatic auf der CCU2 einbinden, schaffe es aber nicht, die Konfiguration der Integration abzuschließen... Ich habe die Dokumentation gelesen, komme aber nicht weiter... Hilfe!... 

Hier einige Infos:
- HomeAssistant mit letzer Version auf Intel NUC
- CCU2 mit Version: 2.61.7 (letzte Version, soweit ich weiss...)
- ich habe keine Änderungen an Firewall oder Ports auf der CCU2 vorgenommen...
- Homematic(IP) Local Integration über HACS
- Autodetection der Integration funktioniert nicht
- ein mannuelles Hinzufügen (mit Standard-Werten für Ports ohne TLS) funktioniert auch nicht, nach klicken von "Absenden" bleibt das Konfigurations-Fenster im Suchmodus, es passiert nichts weiter...

Ich hoffe jemand hat einen Tipp... Vielen Dank!

**Comment by SukramJ** [comment]
Dann liefere doch mal ein paar hilfreiche Informationen:

**Home Assistant:**
- Version: 2022.x.x
- Installation method (HAOS, supervised, core, venv, Docker): 

**Screenshots (if applicable)**
- CCU firewall in CCU

**Relevant HA log entries (press full log) VERY IMPORTANT**

**Reply by MrSplit** [reply]
Danke für die Rückmeldung und sorry für meine Dummheit, die Frage nach den Logs hat schon zur Lösung des Rätsels geführt... 
Ich hatte in der Kofig-Maske den Host-Namen der CCU2 eingetragen, dieser konnte aber nicht aufgelöst werden... Mit der IP ging es dann... 

Sieht soweit alles gut aus, die Geräte wurden alle erkannt, soweit ich sehen kann. 

Danke!

**Reply by SukramJ** [reply]
Danke für die Rückmeldung. Bei Problemen lohnt sich immer ein Blick in die Logs.
